package.json

```diff
@@ -28,7 +28,7 @@
     "react-addons-css-transition-group": "15.1.0",
     "react-addons-shallow-compare": "15.1.0",
     "react-copy-to-clipboard": "4.1.0",
-    "react-dom": "15.1.0",
+    "react-dom": "^15.1.0",
```

Comment:
I think we want to leave the version hardcoded here (e.g. `"react-dom": "15.1.0",`)